cypress/e2e/dashboard/events/events-index.cy.ts

```diff
@@ -16,6 +16,7 @@ const eventData = {
   sponsor_ids: [],
   chapter_id: 1,
 };
+
 describe('events dashboard', () => {
```

Comment:
```suggestion
// TODO: Move these specs into the other describe block, once we can make sure
// that Cypress is operating on an event from chapter 1.
describe('spec needing owner', () => {
```
Just so we know why the specs have been split.